src/Arcus.WebApi.Security/Authorization/Extensions/FilterCollectionExtensions.cs

```diff
@@ -23,9 +24,10 @@ public static FilterCollection AddJwtTokenAuthorization(this FilterCollection fi
         /// </summary>
         /// <param name="filters">All filters that are being applied to the request pipeline</param>
         /// <param name="configureOptions">Configuration options for using JWT token authorization</param>
+        /// <param name="claimCheck">Custom claims key-value pair to validate against</param>
         public static FilterCollection AddJwtTokenAuthorization(
             this FilterCollection filters, 
-            Action<JwtTokenAuthorizationOptions> configureOptions)
+            Action<JwtTokenAuthorizationOptions> configureOptions, IDictionary<string, string> claimCheck = null)
```

Comment:
Can the `claimCheck` argument be null ?  What happens if we pass null ?

Hmm, seems like we're not doing anything with it ?

Reply:
Yes, some tests on this extension and not on the reader would be good too. See if the extension not only guards against invalid input but also configured the reader correctly.

Reply:
done